src/components/Layout/HomeContent.js

```diff
@@ -131,100 +131,104 @@ export function HomeContent() {
             React
           </h1>
           <p className="text-4xl font-display max-w-lg md:max-w-full py-1 text-center text-secondary dark:text-primary-dark leading-snug self-center">
-            The library for web and native user interfaces
+            Pustaka untuk antarmuka pengguna web dan native
           </p>
           <div className="mt-5 self-center flex gap-2 w-full sm:w-auto flex-col sm:flex-row">
             <ButtonLink
               href={'/learn'}
               type="primary"
               size="lg"
               className="w-full sm:w-auto justify-center"
-              label="Learn React">
-              Learn React
+              label="Belajar React">
+              Belajar React
             </ButtonLink>
             <ButtonLink
               href={'/reference/react'}
               type="secondary"
               size="lg"
               className="w-full sm:w-auto justify-center"
-              label="API Reference">
-              API Reference
+              label="Referensi API">
+              Referensi API
             </ButtonLink>
           </div>
         </div>
 
         <Section background="left-card">
           <Center>
-            <Header>Create user interfaces from components</Header>
+            <Header>Membuat antarmuka pengguna dari komponen</Header>
             <Para>
-              React lets you build user interfaces out of individual pieces
-              called components. Create your own React components like{' '}
-              <Code>Thumbnail</Code>, <Code>LikeButton</Code>, and{' '}
-              <Code>Video</Code>. Then combine them into entire screens, pages,
-              and apps.
+              React memungkinkan Anda membangun antarmuka penguna dari
+              bagian-bagian yang disebut komponen. Buat komponen React Anda
+              sendiri seperti <Code>Thumbnail</Code>, <Code>LikeButton</Code>,
+              dan <Code>Video</Code>. Kemudian gabungkan komponen-komponen
+              tersebut ke dalam seluruh layar, halaman, dan aplikasi.
             </Para>
           </Center>
           <FullBleed>
             <Example1 />
           </FullBleed>
           <Center>
             <Para>
-              Whether you work on your own or with thousands of other
-              developers, using React feels the same. It is designed to let you
-              seamlessly combine components written by independent people,
-              teams, and organizations.
+              Baik Anda bekerja sendiri atau dengan ribuan pengembang lain,
+              menggunakan React akan terasa sama. React dirancang untuk
+              memungkinkan Anda menggabungkan komponen yang dirancang oleh
+              pengembang independen, tim, dan organisasi besar secara mulus.
```

Comment:
@resir014 

Should we put the "mulus" in the middle like in the original? Also just "organisasi" seems fine because it also fits for small ones.

```suggestion
              Baik Anda bekerja sendiri atau dengan ribuan pengembang lain,
              menggunakan React akan terasa sama. React dirancang untuk
              memungkinkan Anda menggabungkan komponen secara mulus
              yang dirancang oleh pengembang independen, tim, dan organisasi.
```

Reply:
@mhaidarhanif Yes, let's go with this.